test/quantile_test.lua

```diff
@@ -104,3 +105,26 @@ g.test_package_reload = function()
     local ok, quantile_package = pcall(require, 'metrics.quantile')
     t.assert(ok, quantile_package)
 end
+
+g.test_fiber_yield_stress = function()
+    local q1 = quantile.NewTargeted({[0.5]=0.01, [0.9]=0.01, [0.99]=0.01})
+
+    for _=1,1e6 do
+        t.assert(q1.b_len < q1.__max_samples)
+        quantile.Insert(q1, math.random(1))
+    end
+
+    for _=1,500 do
+        fiber.create(function()
+            for _=1,1e2 do
+                t.assert(q1.b_len < q1.__max_samples)
```

Comment:
is it possible to reduce test execution time with reduced __max_samples? if this is the case this should be done